packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md

```diff
@@ -5,6 +5,19 @@ title: Okta Identity Engine API Products release notes 2022
 
 ## March
 
+### Weekly release 2022.03.1
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Bugs fixed in 2022.03.1](#bugs-fixed-in-2022-03-1) | March 09, 2022 |
+
+#### Bugs fixed in 2022.03.1
+
+* Performing a GET request on the `/policies/` [endpoint](/docs/reference/api/policy/) sometimes returned a page with the wrong number of items. (OKTA-455164)
+* When the [List email templates](/docs/reference/api/brands/#list-email-templates) or the [List email customizations](/docs/reference/api/brands/#list-email-customizations) operations were performed on the `/brands/` endpoint, the base address in the link response header was `-admin` instead of the requested base address. (OKTA-465356)
```

Comment:
```suggestion
* When the [List email templates](/docs/reference/api/brands/#list-email-templates) or the [List email customizations](/docs/reference/api/brands/#list-email-customizations) operations were performed on the `/brands/` endpoint, the base address in the link response header contained the `-admin` string instead of the requested base address. (OKTA-465356)
```